recipes/cargo-make/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+# Script is called by bld.bat by windows shell, which doesn't start here
+cd $SRC_DIR
+cp ${RECIPE_DIR}/rust-toolchain .
+cargo fix --lib -p cargo-make --allow-no-vcs
+cargo build --bins --locked
+cargo build --tests --locked
+cargo test --bins --locked
+cargo install --path . --root ${PREFIX} --locked
```

Comment:
What is the reason that you need more than this last line? That has normally been sufficient for most rust projects.

Reply:
Ok, I will simplify it